README.md

```diff
@@ -49,6 +49,7 @@ Following are the attributes and methods to initialise the demo views.
 |```app:shimmer_demo_layout_manager_type``` | ```setDemoLayoutManager(LayoutManagerType)``` | Layout manager of demo view. Can be one among linear_vertical or linear_horizontal or grid. |
 |```app:shimmer_demo_shimmer_color``` | ``` - ``` | Color reference or value. It can be used to change the color of the shimmer line. |
 |```app:shimmer_demo_angle``` | ``` - ``` | Integer value between 0 and 30 which can modify the angle of the shimmer line. The default value is zero. |
+|```app:shimmer_demo_mask_width``` | ``` - ``` | Float value between 0 and 1 which can modify the width of the shimmer line. The default value is 0.5f. |
```

Comment:
You created a setter method on ShimmerRecyclerView, so for the Java method column the value "-" is not good, replace it with the method name.

Reply:
Also, for the default value please use 0.5 instead of 0.5f since before you mentioned that it is a Float value.